.chloggen/google-sercrets-manager-provider-update-scheme.yaml

```
@@ -0,0 +1,15 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Just because of how it renders in the resulting release changelog, I'd suggest calling this `enhancement`. It wasn't necessarily a bug, and if it was ever released we'd call it `breaking`. In this case we can just call it `enhancement` since it was a change before it ever got released.
```suggestion
change_type: enhancement
```

---

Done